web/themes/joinup/prototype/sass/_mixins.sass

```diff
@@ -72,6 +72,87 @@
     -ms-transform: translate(0, -50%)
     transform: translate(0, -50%)
 
+// Hidden print
 =hidden-print()
   @media print
     display: none !important
+
+// Flexbox
+// Defines a block flex container; enables a flex context for all its direct children.
+=flex
+  display: -webkit-box
+  display: -webkit-flex
+  display: -moz-flex
+  display: -ms-flexbox
+  display: flex
+
+// Inline flex
+// Defines an inline flex container; enables a flex context for all its direct children.
+=inline-flex
+  display: -webkit-inline-box
```

Comment:
Hi @kalinchernev , I have heard about autoprefixer and I found it great, but I wasn't sure if we could use it. However, if this is not a problem then we should start using it as soon as possible. It would be great if you could create a ticket for me to:

- install autoprefixer;
- remove all mixins with prefixes (the new and the old ones);
- clear sass.

Thanks to this we will actually avoid most of our problems related with supporting different browsers / devices.